Configurations/10-main.conf

```diff
@@ -208,7 +208,6 @@ sub vms_info {
         # gcc shared build with Sun C. Given that gcc generates faster
         # code [thanks to inline assembler], I would actually recommend
         # to consider using gcc shared build even with vendor compiler:-)
-        #					<appro@fy.chalmers.se>
```

Comment:
On second thought I'd say that if references are removed, then it might be appropriate to re-word comment. I mean above commentary reads "*I* would actually recommend" and it becomes unclear who is "I". So it would be appropriate to re-word as "it's actually recommended".

Reply:
Put appro@openssl.org